CITATION.cff

```diff
@@ -0,0 +1,123 @@
+# This CITATION.cff file was generated with cffinit.
+# Visit https://bit.ly/cffinit to generate yours today!
+
+cff-version: 1.2.0
+title: GeoServer
+message: >-
+  If you use this software, please cite it using the
+  metadata from this file.
+type: software
+authors:
+  - given-names: Andrea
+    family-names: Aime
+    affiliation: GeoSolutions It
+  - given-names: Alessio
+    family-names: Fabiani
+    affiliation: GeoSolutions It
+  - given-names: Brad
+    family-names: Hards
+  - given-names: Nils
+    family-names: Bühner
+  - given-names: Daniele
+    family-names: Romagnoli
+    affiliation: GeoSolutions It
+  - given-names: Emanuele
+    family-names: Tajariol
+    affiliation: GeoSolutions It
+  - given-names: Fernando
+    family-names: Miño
+    affiliation: GeoSolutions It
+  - given-names: Gabriel
+    family-names: Roldan
+    affiliation: Camptocamp
+  - given-names: Ian
+    family-names: Turton
+  - given-names: James
+    family-names: Hughes
+  - given-names: Jody
+    family-names: Garnett
+    affiliation: GeoCat BV
+    email: jody.garnett@gmail.com
+  - given-names: Josh
+    family-names: Fix
+  - given-names: Jukka
+    family-names: Rahkonen
+  - given-names: Mark
+    family-names: Prins
+    affiliation: B3Partners
+  - given-names: Nuno
+    family-names: Oliveira
+    affiliation: GeoSolutions It
+  - given-names: Peter
+    family-names: Rushforth
+  - given-names: Alessandro
+    family-names: Parma
+    affiliation: GeoSolutions It
+  - given-names: Steve
+    family-names: Ikeoka
+  - given-names: Simone
+    family-names: Giannecchini
+    affiliation: GeoSolutions It
+  - given-names: Kevin
+    family-names: Smith
+    affiliation: Vivid Solutions
+  - given-names: Torben
+    family-names: Barsballe
+    affiliation: Planet Federal
+  - name: GeoSolutions SRL
+    city: Via Carignoni
+    country: IT
+    post-code: '55041'
+    address: Via Carignoni 51
+    location: Camaiore
+    region: Lucca
+    tel: '+390584962313'
+    fax: '+390584660272'
+    website: 'https://www.geosolutionsgroup.com/'
+    email: info@geosolutionsgroup.com
```

Comment:
@simboss do I have address information correct here?

Reply:
Okay got some clarification, "Via Carignoni" was not the city, it translates to "Carignoni Road":

Revised address should be:
```
  - name: GeoSolutions SRL
    city: Camaiore
    country: IT
    post-code: '55041'
    address: Via Carignoni 51
    location: Lucca
    region: Tuscany
```

@ianturton do you wish to update?